cmd/drone-server/drone.go

```diff
@@ -185,6 +185,14 @@ func main() {
 			repo.GET("/logs/:number/:task", server.GetLogs)
 			// repo.POST("/status/:number", server.PostBuildStatus)
 		}
+
+		// Routes for external services
+		repoExternal := repos.Group("")
+		{
+			repoExternal.Use(server.SetRepo())
+
+			repoExternal.GET("/pr/:number", server.GetPullRequest)
```

Comment:
ok, for now, let's just make the path changes (`s/pr/pulls/g`) and I'll separately work on an improved token implementation so that we can combine the two at some point in the future.